docs/installing.md

```diff
@@ -35,6 +74,60 @@ https://docs.microsoft.com/en-us/windows/wsl/install-win10
 [virtualization capabilities]:
 https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x80370102-the-virtual-machine-could-not-be-started-because-a-required-feature-is-not-installed
 
-## Linux
+It is also recommended to have:
+
+- 8 GB of memory
+- 4 CPU
+
+Additional resources may be required depending on the workloads you plan to run.
+
+### Installing Rancher Desktop on Windows
+
+1. Go to the [releases page] on Github.
+1. Find the version of Rancher Desktop you want to download.
+1. Expand the **Assets** section and download the Windows installer. It will be called `Rancher.Desktop.Setup.X.Y.Z.exe`, where `X.Y.Z` is the version of Rancher Desktop.
+1. Navigate to the directory where you downloaded the installer to and run the installer. This will usually be the `Downloads` folder.
+1. Review the License Agreement and click **I Agree** to proceed with the installation.
+1. When the installation completes, click **Finish** to close the installation wizard.
+
+[release page]:
+https://github.com/rancher-sandbox/rancher-desktop/releases
+
+After Rancher Desktop is installed, users will have access to these supporting utilities:
+
+- [Helm](https://helm.sh/)
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/)
+- [nerdctl](https://github.com/containerd/nerdctl)
+- [Kubernetes Image Manager (kim)](https://github.com/rancher/kim)
```

Comment:
This part shouldn't be Windows-specific.

Reply:
Added to other applicable sections as well.